forge-gui/res/cardsfolder/upcoming/bane_lord_of_darkness.txt

```diff
@@ -0,0 +1,13 @@
+Name:Bane, Lord of Darkness
+ManaCost:1 W U B
+Types:Legendary Creature God
+PT:5/2
+S:Mode$ Continuous | Affected$ Card.Self | AddKeyword$ Indestructible | CheckSVar$ CurrentLife | SVarCompare$ LEX | Description$ As long as your life total is less than or equal to half your starting life total, CARDNAME has indestructible.
+SVar:CurrentLife:Count$YourLifeTotal
+SVar:X:Count$YourStartingLife/HalfDown
+T:Mode$ ChangesZone | ValidCard$ Creature.nonToken+Other+YouCtrl | Origin$ Battlefield | Destination$ Graveyard | Execute$ DBAskOpponentDrawOrPlay | TriggerZones$ Battlefield | TriggerDescription$ Whenever another nontoken creature you control dies, target opponent may have you draw a card. If they don't, you may put a creature card with equal or lesser toughness from your hand onto the battlefield.
+SVar:DBAskOpponentDrawOrPlay:DB$ GenericChoice | ValidTgts$ Opponent | Choices$ DBDrawCard,DBCheatCreature
+SVar:DBDrawCard:DB$ Draw | Defined$ You | NumCards$ 1 | SpellDescription$ Controller draws a card
+SVar:DBCheatCreature:DB$ ChangeZone | Origin$ Hand | Destination$ Battlefield | ChangeType$ Creature.toughnessLEY | ChangeNum$ 1 | SpellDescription$ You may put a creature card with equal or lesser toughness from your hand onto the battlefield.
```

Comment:
I think this should this be `Controller may` instead of `You may` for consistency in the choice dialog...